src/logger/src/logger.rs

```diff
@@ -479,16 +382,19 @@ impl fmt::Display for LoggerError {
 /// Implements the "Log" trait from the externally used "log" crate.
 impl Log for Logger {
     // This is currently not used.
-    fn enabled(&self, _metadata: &Metadata) -> bool {
+    fn enabled(&self, _: &Metadata) -> bool {
```

Comment:
nit: I think we can have this method return `unreachable!()` if it isn't used. Same for `flush`.

Reply:
Sounds good. Done.